src/App.js

```diff
@@ -1,12 +1,66 @@
-import React from 'react';
+import React, {useEffect, useState} from "react";
+import axios from 'axios';
 import './App.css';
+import PokemonCards from "./components/pokemon/PokemonCards";
+import Button from "./components/button/Button";
 
-function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+
+// Het resultaat opslaan in state.
+// Een console log maken van het resultaat
+// Bekijk wat je terug krijgt van de request in de dev tools
+
+function  App() {
+    const [pokemon, setPokemon] = useState  ([])
+    const [endpoint, setEndpoint] = useState (`https://pokeapi.co/api/v2/pokemon/jigglypuff`)
+    const [state, setState ]= useState ([])
+
+
+    useEffect ( () => {
+        async function getPokemonData() {
+            try {
+                const result = await axios.get  (`${endpoint}`)
+                console.log ( result.data)
+                setPokemon ( result.data)
+
+            } catch (e) {
+                console.error ( e )
+            }
+        }
+
+        getPokemonData ()
+    }, [endpoint])
+
+    return (
+        <div>
+            <section className="button-bar">
+                <Button
+                    disabled={!pokemon.previous}
```

Comment:
Initieel heeft `pokemon` geen prop `previous`. Als je de default waarde naar `undefined` hebt aangepast kun je hier het volgende doen:
```suggestion
                    disabled={!pokemon || !pokemon.previous}
```